website/templates/base.mako

```diff
@@ -50,6 +50,27 @@
 
 </head>
 <body data-spy="scroll" data-target=".scrollspy">
+
+    % if private_link_anonymous:
+      ## z-index set to 8 to not overlap with getting started with osf non-logged in user bottom-pop-up
+      ## bottom = 30px to not completely block copyright information
+      <style>
```

Comment:
@alexschiller In 8b7674e37195e93eac3d099245ccea37adaf044b we copied this code over to a stylesheet, but we also need to remove it from this location.